modules/ROOT/pages/access-logging.adoc

```diff
@@ -142,3 +142,5 @@ Based on this setting, the NCSA access log will include the following informatio
 * The first line of the request
 * The status code of the response
 * The response size in bytes excluding headers
+* The elapsed time of the request in milliseconds (end-to-end including client and network time)
+* The elapsed time until the first bytes of the response are sent in milliseconds (a close approximation of application response time)
```

Comment:
The elapsed time until the first bytes of the response are sent in milliseconds. This value is a close approximation of application response time.